src/Widgets/Controllers/WidgetsSettingsController.php

```diff
@@ -82,7 +82,7 @@ public function save(): \CodeIgniter\HTTP\RedirectResponse
 
             default:
                 $this->saveWidgetSettings();
-            // dd($this->request->getVar('widget'));
+                // dd($this->request->getVar('widget'));
```

Comment:
We can delete this line.

```suggestion
```

Reply:
This line has been deleted, i just missed it.